fossunited/chapters/doctype/foss_chapter_event/foss_chapter_event.py

```diff
@@ -111,6 +111,25 @@ def before_save(self):
             self.update_published_status()
         self.set_route()
 
+    def validate(self):
+        self.validate_permalink()
+
+    def validate_permalink(self):
+        if self.is_external_event:
+            return
+
+        if frappe.db.exists(
+            self.doctype, {"event_permalink": self.event_permalink}
+        ):
+            frappe.throw(
+                f"Event Permalink {self.event_permalink} already exists!"
+            )
+
+        if "/" in self.event_permalink:
+            frappe.throw(
```

Comment:
With the validation of `/` - should we try for a more stricter validation pattern ? 
If someone can add a permalink like - `f0$$h@(k` - should we make it with alphanumeric chars only ? 

And maybe with a length cap as well too?